test/stream.cc

```diff
@@ -157,6 +163,11 @@ TEST(StreamTest, Fail) {
               });
   };
 
+  static_assert(
+      eventuals::tuple_types_unordered_equals_v<
+          decltype(s())::ErrorsFrom<void, std::tuple<>>,
+          std::tuple<std::exception, std::runtime_error>>);
```

Comment:
Please commit the code before this so we can remove `std::exception` from the resulting list?
```suggestion
          std::tuple<std::runtime_error>>);
```